base64.js

```diff
@@ -37,7 +37,7 @@
   // [https://gist.github.com/1020396] by [https://github.com/atk]
   object.atob || (
   object.atob = function (input) {
-    var str = String(input).replace(/=+$/, '');
+    var str = String(input).replace(new RegExp("=+$"), ''); // #31: ExtendScript bad parse of /=
```

Comment:
If we come to a consensus I can change the commit, but let's wait for @michaelficarra to chime in first.